web/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
```

Comment:
This file should not be modified, please upgrade your `pnpm`.

https://github.com/orgs/pnpm/discussions/4383